newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -33,68 +44,98 @@ export const GameCard = ({
     if (!url) return;
     Window.openExternalURL(url);
   };
-
   return (
     <I18n>
       {({ i18n }) => (
         <Card key={game.id}>
-          <CardHeader
-            title={game.gameName}
-            subheader={
-              <Line alignItems="center" noMargin>
-                <Trans>Created on {i18n.date(game.createdAt * 1000)}</Trans>
-                {isCurrentGame && (
-                  <>
-                    <Spacer />
-                    <Chip
-                      size="small"
-                      label={<Trans>Currently edited</Trans>}
-                      color="primary"
-                    />
-                  </>
-                )}
-                {game.publicWebBuildId && (
-                  <>
-                    <Spacer />
-                    <Chip
-                      size="small"
-                      label={<Trans>Published on Liluo</Trans>}
-                    />
-                  </>
-                )}
-              </Line>
-            }
-          />
-          <CardActions>
-            <ResponsiveLineStackLayout
-              expand
-              noMargin
-              justifyContent="flex-end"
-            >
-              {game.publicWebBuildId && (
-                <RaisedButton
-                  label={<Trans>Open</Trans>}
-                  onClick={openGameUrl}
-                  primary
-                />
-              )}
-              <FlatButton
-                icon={<TuneIcon />}
-                label={<Trans>Details</Trans>}
-                onClick={onOpenDetails}
+          <Line>
+            {game.thumbnailUrl ? (
+              <img
+                src={game.thumbnailUrl}
+                style={{
+                  ...styles.image,
+                  width: thumbnailWidth,
+                  height: thumbnailHeight,
+                  minHeight: thumbnailHeight,
+                }}
+                alt={game.gameName}
+                title={game.gameName}
               />
-              <FlatButton
-                icon={<PlaylistPlayIcon />}
-                label={<Trans>Builds</Trans>}
-                onClick={onOpenBuilds}
+            ) : (
+              <Paper
+                variant="outlined"
+                style={{
+                  width: thumbnailWidth,
```

Comment:
why not add those styles in the `styles` object at the top? same for image styles